app-misc/ddcutil/ddcutil-1.2.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic linux-info udev
+
+DESCRIPTION="Program for querying and changing monitor settings"
+HOMEPAGE="http://www.ddcutil.com/"
+SRC_URI="https://github.com/rockowitz/ddcutil/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="drm introspection usb-monitor user-permissions video_cards_nvidia X"
+REQUIRED_USE="drm? ( X )"
+
+RDEPEND="
+	dev-libs/glib:2
+	sys-apps/i2c-tools
+	virtual/udev
+	drm? ( x11-libs/libdrm )
+	introspection? ( >=dev-libs/gobject-introspection-1.54.0:= )
+	usb-monitor? (
+		dev-libs/hidapi
+		virtual/libusb:1
+		sys-apps/usbutils
+	)
+	user-permissions? (
+		acct-group/i2c
+		usb-monitor? ( acct-group/video )
+	)
+	X? (
+		x11-libs/libXrandr
+		x11-libs/libX11
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+pkg_pretend() {
+	# This program needs /dev/ic2-* devices to communicate with the monitor.
+	CONFIG_CHECK="~I2C_CHARDEV"
+	ERROR_I2C_CHARDEV="You must enable I2C_CHARDEV in your kernel to continue"
+	if use usb-monitor; then
+		CONFIG_CHECK+=" ~HIDRAW ~USB_HIDDEV"
+		ERROR_HIDRAW="HIDRAW is needed to support USB monitors"
+		ERROR_I2C_CHARDEV="USB_HIDDEV is needed to support USB monitors"
+	fi
+
+	# Now do the actual checks setup above
+	check_extra_config
+}
+
+src_prepare() {
+	default
+	eautoreconf
+	sed -e "s#usr/local/bin#usr/bin#" -i data/etc/udev/rules.d/45-ddcutil-usb.rules || die
+}
+
+src_configure() {
+	# Bug 607818.
+	replace-flags -O3 -O2
```

Comment:
Is this still required?

Reply:
Built successful without it. (app-misc/ddcutil-1.2.1  USE="-X -drm -introspection -usb-monitor -user-permissions" VIDEO_CARDS="-nvidia")
Also sed line has dropped from src_prepare.

Reply:
> Built successful without it. (app-misc/ddcutil-1.2.1 USE="-X -drm -introspection -usb-monitor -user-permissions" VIDEO_CARDS="-nvidia") Also sed line has dropped from src_prepare.

Does it build with `CFLAGS="... -O3...." CXXFLAGS="... -O3...."`? Cause that is what this line was working around.

Reply:
> Does it build with `CFLAGS="... -O3...." CXXFLAGS="... -O3...."`? Cause that is what this line was working around.

Yes. My current config from make.conf:
```
COMMON_FLAGS="-march=x86-64-v3 -mtune=generic -O3 -pipe"
CFLAGS="${COMMON_FLAGS}"
CXXFLAGS="${COMMON_FLAGS}"
FCFLAGS="${COMMON_FLAGS}"
FFLAGS="${COMMON_FLAGS}"
```

Reply:
Great, thanks for checking!